R/tt_dotabulation.R

```diff
@@ -2,69 +2,107 @@
 match_extra_args <- function(f,
                             .N_col,
                             .N_total,
+                            .all_col_exprs, 
+                            .all_col_counts,
```

Comment:
It is the nrows of alt_counts_df from all the columns (so you can access them). The impossible split is considered an error and informatively sent back if it happens (there is `.alt_df` and `.alt_df_row`). Otherwise only df counts are used